x-pack/plugins/dataset_quality/public/state_machines/summary_panel/src/notifications.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IToasts } from '@kbn/core/public';
+import { i18n } from '@kbn/i18n';
+
+export const fetchDatasetsQualityFailedNotifier = (toasts: IToasts, error: Error) => {
+  toasts.addDanger({
+    title: i18n.translate('xpack.datasetQuality.fetchDatasetsQualityDetailsFailed', {
+      defaultMessage: "We couldn't get your datasets quality details. Default values are shown.",
```

Comment:
Should we show for example `0` the default value or something like `-` ?

Reply:
I'd prefer to put `-` and remove the last part of the error message but maybe @isaclfreire could give better insights